File: app/src/main/java/com/brianmk/exposurenotes/FilmData.java
package com.brianmk.exposurenotes;

public class FilmData {
    private static final String LOG_TAG = FilmData.class.getSimpleName();


    private String film;
    private String iso;
    private String dev;

    public FilmData() {
        this.film = "";
        this.iso = "";
        this.dev = "0";
    }


    public FilmData(String c, String l, String f, String i, String d) {
        this.film = f;
        this.iso = i;
        this.dev = d;
    }

    public void setFilm(String f) { this.film = f; }
    public String getFilm() { return this.film; }

    public void setIso(String i) { this.iso = i; }
    public String getIso() { return this.iso; }

    public void setDev(String d) { this.dev = d; }
    public String getDev() { return this.dev; }
}

File: app/src/main/java/com/brianmk/exposurenotes/FilmData.kt
package com.brianmk.exposurenotes

data class FilmData(var manu: String = "",
                    var name: String = "",
                    var isoIdx: Int = 5,
                    var frames: Int = 0,
                    var devIdx: Int = 4,
                    var notes: String = "") {

    private val isos = arrayOf("0", "5", "12", "25", "50", "100", "160", "200", "400", "800", "1600", "3200", "6400")
    private val devs = arrayOf("0", "-3", "-2", "-1", "0", "+1", "+2", "+3")

    var iso = isos[isoIdx]
    var dev = devs[devIdx]

    fun updateData() {
        iso = isos[isoIdx]
        dev = devs[devIdx]
    }

    companion object {
        private val LOG_TAG = FilmData::class.java.simpleName
    }
}
